src/frontend/src/pages/ChangeRequestsPage/ChangeRequestsOverview.tsx

```diff
@@ -79,55 +77,36 @@ const ChangeRequestsOverview: React.FC = () => {
     )
     .sort((a, b) => (a.dateReviewed && b.dateReviewed ? b.dateReviewed.getTime() - a.dateReviewed.getTime() : 0));
 
-  const displayCRCards = (crList: ChangeRequest[]) => (
-    <Box
-      sx={{
-        display: 'flex',
-        flexDirection: 'row',
-        flexWrap: 'wrap',
-        overflow: 'auto',
-        justifyContent: 'flex-start',
-        '&::-webkit-scrollbar': {
-          height: '20px'
-        },
-        '&::-webkit-scrollbar-track': {
-          backgroundColor: 'transparent'
-        },
-        '&::-webkit-scrollbar-thumb': {
-          backgroundColor: theme.palette.divider,
-          borderRadius: '20px',
-          border: '6px solid transparent',
-          backgroundClip: 'content-box'
-        }
-      }}
-    >
-      {crList.map((cr: ChangeRequest) => (
-        <ChangeRequestDetailCard changeRequest={cr}></ChangeRequestDetailCard>
-      ))}
-    </Box>
-  );
+  const crToReviewRow = {
+    title: 'To Review',
+    changeRequests: crToReview,
+    noChangeRequestsMessage: 'No change requests to review'
+  };
+  const crUnreviewedRow = {
+    title: 'My Un-reviewed Change Requests',
+    changeRequests: crUnreviewed,
+    noChangeRequestsMessage: 'No un-reviewed change requests'
+  };
 
-  const renderChangeRequests = (title: string, crList: ChangeRequest[], emptyMessage: string) => {
-    return (
-      <>
-        <Typography variant="h5" gutterBottom>
-          {title}
-        </Typography>
-        {crList.length > 0 ? (
-          <Grid container>{displayCRCards(crList)}</Grid>
-        ) : (
-          <Typography gutterBottom>{emptyMessage}</Typography>
-        )}
-      </>
-    );
+  const crApprovedRow = {
+    title: 'My Recently Approved Change Requests',
+    changeRequests: crApproved,
+    noChangeRequestsMessage: 'No recently approved change requests'
   };
 
+  const overviewRowList = [crUnreviewedRow, crApprovedRow];
+  if (showToReview) overviewRowList.unshift(crToReviewRow);
+
   return (
-    <Box>
-      {showToReview && renderChangeRequests('To Review', crToReview, 'No change requests to review')}
-      {renderChangeRequests('My Un-reviewed Change Requests', crUnreviewed, 'No un-reviewed change requests')}
-      {renderChangeRequests('My Recently Approved Change Requests', crApproved, 'No recently approved change requests')}
-    </Box>
+    <>
+      {overviewRowList.map((crRow) => (
+        <ChangeRequestRow
+          title={crRow.title}
+          changeRequests={crRow.changeRequests}
+          noChangeRequestsMessage={crRow.noChangeRequestsMessage}
+        />
```

Comment:
Same comment here about not doing the map, and just conditionally render the crToReview row only if showToReview is true. I think this'll make it a bit more clear what information is going where